lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -233,33 +243,24 @@ defmodule MembraneLiveWeb.EventChannel do
 
       {:ok, is_presenter} = check_if_presenter(email, true, id)
 
-      new_offset =
-        if is_auth and is_presenter do
-          case :ets.lookup(:start_timestamps, id) do
-            [{_key, start_timestamp}] ->
-              System.monotonic_time(:millisecond) - start_timestamp
+      key = if is_auth and is_presenter, do: :start_timestamps, else: :client_start_timestamps
 
-            #  TODO handle messages when presenter is in the "start streaming" window
-            [] ->
-              raise "Recieved chat message from presenter when there is no presenters connected to RTC Engine process"
-          end
```

Comment:
So in a situation like this will we simply send messages without delay?

Reply:
Before such case would indicate that something is broken, but now one can be a 'presenter' and not stream anything through `hls`, so I believe that this is appropriate.